aries-site/src/examples/templates/page-layouts/components/PageContainer.js

```diff
@@ -0,0 +1,59 @@
+import { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Box, ResponsiveContext } from 'grommet';
+
+// Theme-like object specifying alignment, width, and spacing for
+// a PageContainer.
+export const pageContainer = {
+  wide: {
+    align: 'center',
+    width: {
+      max: 'xxlarge', // 1536 --> needs to adjust to 1536 + 72
+    },
+  },
+  narrow: {
+    align: 'center',
+    width: {
+      max: 'large', // 768
+    },
+  },
+  full: {
+    align: 'start',
+    width: {
+      max: '100%',
+    },
+  },
+  gap: {
+    xsmall: 'medium',
+    small: 'medium',
+    medium: 'medium',
+    large: 'medium',
+    xlarge: 'medium',
+  },
+  pad: {
+    xsmall: { horizontal: 'medium' },
+    small: { horizontal: 'large' },
+    medium: { horizontal: 'medium' },
+    large: { horizontal: 'large' },
+    xlarge: { horizontal: 'large' },
+  },
+};
+
+export const PageContainer = ({ kind = 'wide', ...rest }) => {
+  const size = useContext(ResponsiveContext);
+
+  return (
+    <Box
```

Comment:
Doesn't have to be solved in this PR, but thinking about if it would be helpful to have a `PageContainerContext` that held the `pad` value / anything else that would be relevant for components down the line. 

```
export const PageContainer = (...) => {
   const size = useContext(ResponsiveContext):
   const pad = pageContainer.pad[size];
   return (
      <PageContainerContext.Provider value={{ pad }}>
         <Box
           ...
         />
      </PageContainerContext.Provider>
```

Then, in PageContainerWide above, we could do something like the below, as opposed to having to pull from a theme object:
```
const { pad } = useContext(PageContainerContext);
```

Thinking ahead more to if we were to make these things consumable helper components, not critical to this PR merging though.

Reply:
I was having this thought last night as well. I think this ☝️  is the direction we should go.

Reply:
Done.